brax/v2/base.py

```diff
@@ -335,12 +338,13 @@ class Box(Geometry):
   """
 
   halfsize: jp.ndarray
+  rgba: Optional[jp.ndarray] = None
 
 
 @struct.dataclass
 class Plane(Geometry):
   """An infinite plane whose normal points at +z in its coordinate space."""
```

Comment:
could you update the docstrings to something like, `rgba: (4,) the rgba to display in the renderer`